src/main/asciidoc/reference-guide/tutorials/configdatabaseresource.adoc

```diff
@@ -1,14 +1,7 @@
 === Configure a Database resource
-:hardbreaks:
 
 For Syncope 2.0.x
```

Comment:
Please remove this al well: the Reference Guide is for a specific version of Apache Syncope - currently, 2.0.5-SNAPSHOT